examples/wallet_smart_contract/src/main.sw

```diff
@@ -1,19 +1,22 @@
-// NOTE: Storage is a work in progress (see
-// https://github.com/FuelLabs/sway/pull/646), but once it is implemented,
-// declaring storage should look like this.
-
 contract;
 
-use std::*;
-use std::address::Address;
-use std::assert::assert;
+use std::{
+    address::Address, 
+    assert::assert, 
+    chain::auth::{AuthError, Sender, msg_sender}, 
+    constants::NATIVE_ASSET_ID, 
+    context::{call_frames::msg_asset_id, msg_amount}, 
+    contract_id::ContractId, 
+    panic::panic, 
+    result::*, 
+    token::transfer_to_output
+};
 
 const OWNER_ADDRESS: b256 = 0x8900c5bec4ca97d4febf9ceb4754a60d782abbf3cd815836c1872116f203f861;
-const NATIVE_ASSET_ID: b256 = 0x0000000000000000000000000000000000000000000000000000000000000000;
 
-// storage {
-//     balance: u64,
-// }
+storage {
+    balance: u64, 
```

Comment:
Is this file formatted?

Reply:
It actually. I tried again but the space after `u64,` keeps showing up. I'll file an issue.